test/testUtils/mayaUtils.py

```diff
@@ -29,6 +29,7 @@
     pass
 
```

Comment:
Removed some unused methods, added `openNewScene`, and added `applyTestSettings`, which mostly applies the test settings that were previously in `snapshot`.